stdlib/public/core/VarArgs.swift

```diff
@@ -467,6 +491,140 @@ final internal class _VaListBuilder {
   internal var storage: ContiguousArray<Int>
 }
 
+#elseif arch(arm64) && !(os(macOS) || os(iOS) || os(tvOS) || os(watchOS))
+
+
+@_fixed_layout // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
+final internal class _VaListBuilder {
+
+  @_inlineable // FIXME(sil-serialize-all)
+  @_versioned // FIXME(sil-serialize-all)
+  internal init() {
+    // prepare the register save area
+    allocated = _registerSaveWords
+    storage = allocStorage(wordCount: allocated)
+    // append stack arguments after register save area
+    count = allocated
+  }
+
+  @_inlineable // FIXME(sil-serialize-all)
+  @_versioned // FIXME(sil-serialize-all)
+  deinit {
+    if let allocatedStorage = storage {
+      deallocStorage(wordCount: allocated, storage: allocatedStorage)
+    }
+  }
+
+  @_inlineable // FIXME(sil-serialize-all)
+  @_versioned // FIXME(sil-serialize-all)
+  internal func append(_ arg: CVarArg) {
+    var encoded = arg._cVarArgEncoding
+
+    if arg is _CVarArgPassedAsDouble
+      && fpRegistersUsed < _countFPRegisters {
+        var startIndex = (fpRegistersUsed * _fpRegisterWords)
```

Comment:
Indenting should be increased by two spaces compared to “if”—you can choose to move the brace down to its own line when the conditional expression is multi-line, if that helps you.